README.md

```diff
@@ -573,6 +574,9 @@ func main() {
 	if Logger, err = sdklogger.NewBuilder(loggerConfig).SetTracking(trackingConfig).Build(); err != nil {
 		log.Fatalf("Failed to load SDK logger: %s", err.Error())
 	}
+
+	err = errors.New("new error")
+	logger.WithError(err).Errorf("trying sentry error functionality")
```

Comment:
[q]: can we also add `WithError` to all `Fatal` calls?

In a separate commit, we can update all occurrences of `log.Fatal` to include `WithError(err)` in the README., wdyt?

Reply:
I don't understand the change request. "add this to all fatal calls"? can you suggest change?

Reply:
@laynax Sorry, let me elaborate here. In the README.md file for this section, we have a couple of [log.Fatalf calls](https://github.com/scribd/go-sdk/blob/c8fad6a2f362691f16e4bd27e0aec016a0f25f11/README.md?plain=1#L570-L574). `Fatalf` indicates a major failure that, probably, should be tracked by Sentry.

But now I see that those `Fatalf` calls are from the `log` package. Then, you can ignore my question here 😃 

>I don't understand the change request

Nah, that was not a change request, it is just a comment/question 👍